### PR TITLE
scripts: install runc if not found in get script

### PR DESCRIPTION
**Pull request by haircommander**
Signed-off-by: Peter Hunt <pehunt@redhat.com>

#### What type of PR is this?

/kind ci
#### Does this PR introduce a user-facing change?

```release-note
none
```

### CODE REVIEW

File: contrib/bundle/build [hunk 1]
```diff
@@ -162,8 +160,8 @@ pushd $TAR_DIR
 export DESTDIR=test/
 ./install
 EXP_CNT=64
-if [[ $ARCH == "$ARCH_AMD64" ]]; then
-    EXP_CNT=65
+if ! command -v runc; then
```
Comment (haircommander) [inline contrib/bundle/build:163]: @saschagrunert is this still necessary? should we unconditionally install?

**Comment by haircommander** [comment]
/hold
carrying https://github.com/cri-o/cri-o/pull/5506 to get runc 1.1-rc, which does define an arm binary, whereas older version did not

File: contrib/bundle/build (hunk 1 above)
Comment (saschagrunert) [reply contrib/bundle/build:163]: I think we should install it in any case, yes.

**Comment by TomSweeneyRedHat** [comment]
LGTM
assuming happy tests.

**Comment by saschagrunert** [comment]
A friendly reminder that this PR had no activity for 30 days.

**Review by saschagrunert — APPROVED** [review]
/retest
/lgtm

**Pull request merged**